README.md

```diff
@@ -61,25 +61,29 @@ import (
 	"io/ioutil"
 	"log"
 
-	"github.com/emersion/go-smtp"
+	smtp "github.com/emersion/go-smtp"
```

Comment:
Is this needed by `go vet`?

Reply:
Probalby not. This change was automatically made by the Go tools used by Visual Studio Code.

Reply:
Can you remove this change and rebase your PR?